app/move/fields/create.js

```diff
@@ -336,6 +336,40 @@ module.exports = {
       text: 'fields::documents.hint',
     },
   },
+  move_agreed: {
+    validate: 'required',
+    component: 'govukRadios',
+    name: 'move_agreed',
+    fieldset: {
+      legend: {
+        text: 'fields::move_agreed.label',
+        classes: 'govuk-visually-hidden govuk-fieldset__legend--m',
+      },
+    },
+    items: [
+      {
+        id: 'move_agreed',
+        value: 'yes',
+        text: 'Yes',
+        conditional: 'move_agreed_by',
+      },
+      {
+        value: 'no',
+        text: 'No',
+      },
+    ],
+  },
+  move_agreed_by: {
+    id: 'move_agreed_by',
```

Comment:
Just spoke to @Andy-Millar and this should be required if they answer `true`.

You'll need to add the `dependent` property too:

```javascript
move_agreed_by: {
  ...
  validate: 'required',
  dependent: {
    field: 'move_agreed',
    value: 'yes',
  }
  ...
}
```